src/proj.h

```diff
@@ -328,6 +340,17 @@ extern const PJ      *proj_shutdown;
 #endif
 
 
+/* Set or read error level */
+int  proj_errno (PJ *P);
+void proj_errno_set (PJ *P, int err);
```

Comment:
They are quite useful for reporting from application specific functions, that take `PJ` arguments: In the main program one just need to do one kind of error checking, not having to care whether an error was reported from the code in `function_doing_my_proj_stuff(PJ *P, ...)` or in the API code called by that function.

`pj_ctx_set_errno()` is also part of `proj_api.h` - I assume for the same reason.